Docs/Book/RDKit_Book.rst

```diff
@@ -1498,7 +1498,7 @@ Here's a sample block from an SDF that demonstrates all of the features, they ar
   >  <atom.iprop.NumHeavyNeighbors>  (1)
   2 2 2
 
-  >  <atom.prop.AtomLabel>  (1)
+  >  <atom.prop.AtomSymbol>  (1)
```

Comment:
Is this a result of automatic refactoring gone wrong?

Reply:
Sorry, yes.  I will pay more attention to a re-factor in future.  I didn't realise CLion would try so hard, so far outside where I was working.